packages/office-ui-fabric-react/src/components/Nav/Nav.Props.ts

```diff
@@ -173,6 +183,12 @@ export interface INavLink {
   parentId?: string;
 
   /**
+   * (Optional) By default, any link with onClick defined will render as a button.
+   * Set this property to true to override that behavior.
+   */
+  isAnchor?: boolean;
```

Comment:
I changed the name to `forceAnchor` and added a comment about the default behavior.